src/pages/settings/Subscription/RequestEarlyCancellationPage/index.tsx

```diff
@@ -0,0 +1,116 @@
+        <Text>
+            {translate('subscription.requestEarlyCancellation.acknowledgement.part1')}
+            <TextLink href={CONST.TERMS_URL}>{translate('subscription.requestEarlyCancellation.acknowledgement.link')}</TextLink>
+            {translate('subscription.requestEarlyCancellation.acknowledgement.part2')}
+        </Text>
+    );
+
+    let screenContent: React.ReactNode;
+
+    if (cancellationType === CONST.CANCELLATION_TYPE.MANUAL) {
+        screenContent = (
+            <View style={[styles.flexGrow1, styles.justifyContentBetween, styles.mh5]}>
+                <View>
+                    <Text style={styles.textHeadline}>{translate('subscription.requestEarlyCancellation.requestSubmitted.title')}</Text>
+                    <Text style={[styles.mt1, styles.textNormalThemeText]}>
+                        {translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.part1')}
+                        <TextLink onPress={() => Report.navigateToConciergeChat()}>{translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.link')}</TextLink>
+                        {translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.part2')}
+                    </Text>
+                </View>
+                <FixedFooter>
+                    <Button
+                        success
+                        text={translate('subscription.requestEarlyCancellation.submitButton')}
+                        onPress={() => Navigation.goBack()}
+                    />
+                </FixedFooter>
+            </View>
+        );
+    } else if (cancellationType === CONST.CANCELLATION_TYPE.AUTOMATIC) {
+        screenContent = (
+            <View style={[styles.flexGrow1, styles.justifyContentBetween, styles.mh5]}>
+                <View>
+                    <Text style={styles.textHeadline}>{translate('subscription.requestEarlyCancellation.subscriptionCancelled.title')}</Text>
+                    <Text style={[styles.mt1, styles.textNormalThemeText]}>{translate('subscription.requestEarlyCancellation.subscriptionCancelled.subtitle')}</Text>
+                    <Text style={[styles.mv4, styles.textNormalThemeText]}>{translate('subscription.requestEarlyCancellation.subscriptionCancelled.info')}</Text>
+                    <Text>
+                        {translate('subscription.requestEarlyCancellation.subscriptionCancelled.preventFutureActivity.part1')}
+                        <TextLink onPress={() => Navigation.navigate(ROUTES.SETTINGS_WORKSPACES)}>
+                            {translate('subscription.requestEarlyCancellation.subscriptionCancelled.preventFutureActivity.link')}
+                        </TextLink>
+                        {translate('subscription.requestEarlyCancellation.subscriptionCancelled.preventFutureActivity.part2')}
+                    </Text>
+                </View>
+                <FixedFooter>
+                    <Button
+                        success
```

Comment:
I haven't checked the view yet but should we set the size as `large`?

Reply:
Definitely, thanks!